tests/node_compat/run_all_test_unmodified.ts

```diff
@@ -336,7 +336,7 @@ async function main() {
   );
   console.log(`Elapsed time: ${((Date.now() - start) / 1000).toFixed(2)}s`);
   // Store the results in a JSON file
-  Deno.writeTextFile(
+  await Deno.writeTextFile(
```

Comment:
This error seems causing linux report cut off in the middle (windows and darwin reports seem created correctly even with this bug..)